resync/api/agent_evolution_api.py

```
@@ -179,9 +179,8 @@ async def submit_feedback(request: SubmitFeedbackRequest):
         )
 
     except Exception as e:
-        # Re-raise programming errors — these are bugs, not runtime failures
-        if isinstance(e, (TypeError, KeyError, AttributeError, IndexError)):
-            raise
+        # FIX: Let global exception handler deal with errors properly
+        # Don't re-raise programming errors - let them propagate to global handler
         logger.error("Failed to collect feedback: %s", e, exc_info=True)
```

Ruff (0.15.1)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)